supply/lib/supply/uploader.rb

```diff
@@ -147,11 +153,7 @@ def upload_bundles
       return unless aab_path
 
       UI.message("Preparing aab at path '#{aab_path}' for upload...")
-      apk_version_codes = [client.upload_bundle(aab_path)]
-
-      # Only update tracks if we have version codes
-      # Updating a track with empty version codes can completely clear out a track
-      update_track(apk_version_codes) unless apk_version_codes.empty?
+      return [client.upload_bundle(aab_path)]
```

Comment:
Isn't updating tracks still necessary here? (Random drive by comment, but I'm curious as to why this function differs from the logic in the newly split upload_apks)

Reply:
Same thoughts/concern there ^ 

@tmtrademarked Thanks for drive by commenting 😊

Reply:
The original logic sequence is below.

### If uploading binary is apks
1. Upload apks
2. Upload mapping files
3. Update track
4. Upload bundle (not executed)
5. Update track (not executed because of empty apk version codes)

### If uploading binary is bundle
1. Upload apks (not executed)
2. Upload mapping (error caused by empty apk version codes)
3. Update track (not executed because of empty apk version codes)
4. Upload bundle
5. Update track

Update track should be executed after upload apks or bundle and only once.
So, I split upload mapping and updating track function to execute after uploading both apk and bundle.

Reply:
After fixing,

1. Upload apks if upload type is apks
2. Upload bundle if upload type is bundle
3. Upload mapping
4. Update track

I think that uploading logic order is not changed 😃

Reply:
Thank you comments and I agree with you 😄
 I fixed at b7e91f0!

Reply:
@joshdholtz Could you review this?